crystal_diffusion/data/parse_lammps_outputs.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import os
+
+import pandas as pd
+import yaml
+
+
+def parse_lammps_output(lammps_dump: str, lammps_thermo_log: str, output_name: str):
+    """Parse a LAMMPS output file and save in a .csv format.
+
+    Args:
+        lammps_dump: LAMMPS output file
+        lammps_thermo_log: LAMMPS
+        output_name: name of parsed output written by the script
+    """
+    if not os.path.exists(lammps_dump):
+        raise ValueError(f'{lammps_dump} does not exist. Please provide a valid LAMMPS dump file as yaml.')
+
+    if not os.path.exists(lammps_thermo_log):
+        raise ValueError(f'{lammps_thermo_log} does not exist. Please provide a valid LAMMPS thermo log file as yaml.')
+
+    # get the atom information (positions and forces) from the LAMMPS 'dump' file
+    with open(lammps_dump, 'r') as f:
+        dump_yaml = yaml.safe_load_all(f)
+        # every MD iteration is saved as a separate document in the yaml file
```

Comment:
If `dump_yaml` contains all the data from the file, there is no need for the rest of the code block to be indented within the `with` block.